content/blog/2022-04-14-april-22-heartbeat.md

```diff
@@ -0,0 +1,339 @@
+---
+title: April '22 Heartbeat
+date: 2022-04-14
+description: >
+  Monthly updates are here! You will find the future of AI Infrastruture is
+  modular, articles on distribution drift and how to solve it, the usual great
+  tutorials and workflows from the Community,  online course updates, new docs
+  and more! Happy April!
+
+descriptionLong: |
+  This month you will find:
+
+    🧱 AI infrastructure is becoming modular,
+
+    🔎 Distribution drift with Chip Huyen
+    
+    🥰 Tutorials and workflows from the Community,
+
+    🗣 IRL events,
+
+    💻 Online course(s) updates,
+
+    🚀 Quebec becoming an Iterative hub, and more!
+picture: 2022-04-14/april-heartbeat-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-22-hearbeat/1158
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - Git
+  - Chip Huyen
+  - Distribution drift
+  - AI Environments
+---
+
+<details>
+
+This month's Heartbeat image is inspired by Community member Gudmundur
+Heimisson. Gudmundur submitted some great PRs to update WebHDFS docs pending
+some other issues in the DVC repo.
+
+This image refelcts his Paris area team's view of Château de Vincennes out their
+company windows!
+
+We are grateful for all our Community members' contributions from all around the
+world!
+
+<summary>✨Image Inspo✨</summary>
+</details>
+
+## AI News
+
+https://media.giphy.com/media/l0JMrPWRQkTeg3jjO/giphy.gif
```

Comment:
looks a bit too heavy. start with some text first? Otherwise it's an image and then this gif - you have to scroll to get to the start.

Reply:
Ok.  Ya.  See what you mean. Adding some text to set up the building blocks...